src/markerclusterer.js

```diff
@@ -1672,3 +1672,5 @@ if (typeof String.prototype.trim !== 'function') {
   };
 }
 
+module.exports = MarkerClusterer;
```

Comment:
I didn't make that edit so idk the justification, I simply merged @dpellier's work. I'll fix it either way. Not going to go full UMD style though, that seems too invasive for this pr.